src/main/java/codersafterdark/reskillable/loot/LootConditionRequirement.java

```diff
@@ -0,0 +1,79 @@
+package codersafterdark.reskillable.loot;
+
+import codersafterdark.reskillable.Reskillable;
+import codersafterdark.reskillable.api.data.PlayerDataHandler;
+import codersafterdark.reskillable.api.data.RequirementHolder;
+import codersafterdark.reskillable.lib.LibMisc;
+import com.google.gson.*;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.JsonUtils;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.LootContext;
+import net.minecraft.world.storage.loot.conditions.LootCondition;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Random;
+import java.util.stream.StreamSupport;
+
+public class LootConditionRequirement implements LootCondition {
+    private final RequirementHolder requirementHolder;
+    private final boolean requiresPlayer;
+    private final String[] requirements;
+
+    public LootConditionRequirement(boolean requiresPlayer, String[] requirements) {
+        this.requiresPlayer = requiresPlayer;
+        this.requirementHolder = requirements.length > 0 ? RequirementHolder.fromStringList(requirements) :
+                RequirementHolder.realEmpty();
+        this.requirements = requirements;
+    }
+
+    @Override
+    @ParametersAreNonnullByDefault
+    public boolean testCondition(Random rand, LootContext context) {
```

Comment:
This should potentially obey the two config options for whether or not to enforce requirements on Fake Players and Creative Players. Not really needed but would probably be good for consistency.

Reply:
I'll leave it in, but if it's blank, I'll default to the configs.